fzf-zsh-plugin.plugin.zsh

```diff
@@ -67,24 +67,25 @@ fi
 # Don't step on user's FZF_DEFAULT_COMMAND
 if [[ -z "$FZF_DEFAULT_COMMAND" ]]; then
   export FZF_DEFAULT_COMMAND='find . -type f ( -path .git -o -path node_modules ) -prune'
+  export FZF_ALT_C_COMMAND='find . -type d ( -path .git -o -path node_modules ) -prune'
 
   if has rg; then
     # rg is faster than find, so use it instead.
     export FZF_DEFAULT_COMMAND='rg --files --hidden --follow --glob "!{.git,node_modules}/**"'
   fi
 
   # If fd command is installed, use it instead of find
-  if has 'fd'; then
+  if has 'fd' || has 'fdfind'; then
```

Comment:
Should the `fdfind` case be an alternate stanza that sets `FZF_DEFAULT_COMMAND` and `FZF_ALT_C_COMMAND` to use `fdfind` instead of `fd`?

Reply:
Yeah I think I missed a bit of detail here. Let me explain a bit.

On Debian, `fd` is intalled as `fdfind` because there is already a command called `fd`. The official solution is to [include an extra `PATH`](https://salsa.debian.org/rust-team/debcargo-conf/-/blob/master/src/fd-find/debian/README.Debian).

I personally don't like polluting my `$PATH` so I `alias fd=fdfind`. But this happens after I load this plugin so it can't detect `fd` yet. I will add some code to use `fdfind`.